### DISCUSSION TITLE
Unstable reading.

### DISCUSSION THREAD

**Discussion by bucklevision** [jomjol/AI-on-the-edge-device#2720] [General]
I'm having issues with my numeric values not being properly detected, I've done a ton to reduce reflection and get alignment correct but even with 0.1 resolution enabled, it skips up and down and loses track after a day. I think part of my problem is my meter seems to go the opposite way from what is expected when progressing normally, but it also does move backward when generating, but I'm losing the will to live at this stage. The black digits read out continuously, the red/white side digits are the decimals which I'd love to read but are too small and fuzzy even from 200mm away.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/29090176/95dd70d3-2edc-4b6a-acf2-d843b924714f)

**Comment by caco3** [comment]
Why are you 20cm away with the cam? Try to go as close as possible so you still have all digits in the picture.
And please provide a screenshot of the Recognition page.

Turning backward (decreasing the value) is not a problem, but you must uncheck the neg. rate check.

**Comment by bucklevision** [comment]
Thanks for the reply! 

Digits are quite a way into the meter so the closer I get the more they're affected by the depth of the socket they're in and the off-perpendicular viewing angle. I could try to get closer for sure, though getting a semi pleasant lighting from the onboard LED was hard enough this far away with diffuser and reflector materials!

I do have the neg rate unchecked 👍

![Screenshot_20231201_214506_Chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/29090176/16c7fc00-69ad-4fb5-af2f-cea36074825e)
![Screenshot_20231201_214524_Chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/29090176/6786cb7d-67d7-4e9a-879c-4a38d987adb3)
![Screenshot_20231201_214549_Chrome](https://github.com/jomjol/AI-on-the-edge-device/assets/29090176/0924f1ea-f616-4b14-9d6b-6520dde6dac0)

**Reply by caco3** [reply]
You could try with external lighting, see https://jomjol.github.io/AI-on-the-edge-device-docs/External-LED/
If you can, try to increase the contrast.
And the reference marks should be as far apart as possible for better alignment.

Can you provide a DEBUG log?
